### ISSUE TITLE
Lack of error description

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1763]
When attempting to submit a form without picking an option on how to answer the form, you get the generic "Error" message.

**Comment by dariiing** [comment]
> > When attempting to submit a form without picking an option, you get the generic "Error" message.
> 
> [@dariiing](https://github.com/dariiing) can you specify which form?

Hello,
This occurs with any type of form.
